scripts/loadable_app/hello.c

```diff
@@ -0,0 +1,12 @@
+int a = 9;
+int b = 9;
+
+int x()
+{
+return 7;
+}
+int main(int argc, char **argv)
+{
+	int p = x();
+	return a + p + x();
```

Comment:
`080480b8 <x>:
 80480b8:	55                   	push   %ebp
 80480b9:	89 e5                	mov    %esp,%ebp
 80480bb:	b8 07 00 00 00       	mov    $0x7,%eax
 80480c0:	5d                   	pop    %ebp
 80480c1:	c3                   	ret    

080480c2 <main>:
 80480c2:	55                   	push   %ebp
 80480c3:	89 e5                	mov    %esp,%ebp
 80480c5:	56                   	push   %esi
 80480c6:	53                   	push   %ebx
 80480c7:	83 e4 f0             	and    $0xfffffff0,%esp
 80480ca:	83 ec 10             	sub    $0x10,%esp
 80480cd:	e8 2c 00 00 00       	call   80480fe <__x86.get_pc_thunk.bx>
 80480d2:	81 c3 a6 00 00 00    	add    $0xa6,%ebx
 80480d8:	e8 db ff ff ff       	call   80480b8 <x>
 80480dd:	89 44 24 0c          	mov    %eax,0xc(%esp)
 80480e1:	8d 83 0c 00 00 00    	lea    0xc(%ebx),%eax
 80480e7:	8b 10                	mov    (%eax),%edx
 80480e9:	8b 44 24 0c          	mov    0xc(%esp),%eax
 80480ed:	8d 34 02             	lea    (%edx,%eax,1),%esi
 80480f0:	e8 c3 ff ff ff       	call   80480b8 <x>
 80480f5:	01 f0                	add    %esi,%eax
 80480f7:	8d 65 f8             	lea    -0x8(%ebp),%esp
 80480fa:	5b                   	pop    %ebx
 80480fb:	5e                   	pop    %esi
 80480fc:	5d                   	pop    %ebp
 80480fd:	c3                   	ret    

080480fe <__x86.get_pc_thunk.bx>:
 80480fe:	8b 1c 24             	mov    (%esp),%ebx
 8048101:	c3                   	ret    
`